modules/search-service-impl/src/main/java/org/opencastproject/search/endpoint/SearchRestService.java

```diff
@@ -146,6 +147,30 @@ public Response remove(@PathParam("id") String mediaPackageId) throws SearchExce
     }
   }
 
+  @DELETE
+  @Path("/seriesId/{seriesid}")
+  @Produces(MediaType.APPLICATION_XML)
+  @RestQuery(name = "removeSeries", description = "Removes a series from the search index, if there are no episodes left connected to the series.", pathParameters = { @RestParameter(description = "The series ID to remove from the search index.", isRequired = true, name = "seriesid", type = RestParameter.Type.STRING) }, responses = {
+          @RestResponse(description = "The removing job.", responseCode = HttpServletResponse.SC_OK),
+          @RestResponse(description = "There has been an internal error and the series could not be deleted", responseCode = HttpServletResponse.SC_INTERNAL_SERVER_ERROR) }, returnDescription = "The job receipt")
+  public Response removeSeries(@PathParam("seriesid") String seriesId) throws SearchException {
+    try {
+      SearchQuery searchQuery = new SearchQuery();
+      searchQuery.withText(seriesId);
+      SearchResult searchResult = searchService.getForAdministrativeRead(searchQuery);
+      if (searchResult.size() > 0) {
+        logger.error("Can not delete series: '{}', it still contains episodes.", seriesId);
+        return Response.serverError().build();
+      }
+
+      Job job = searchService.deleteSeries(seriesId);
```

Comment:
I'm curious, why use a `Job` and do this asynchronously? Shouldn't the delete operation be fast enough? Like this, users will not get any errors.